yarn.lock

```diff
@@ -20256,11 +20316,16 @@ prepend-http@^2.0.0:
   resolved "https://registry.yarnpkg.com/prepend-http/-/prepend-http-2.0.0.tgz#e92434bfa5ea8c19f41cdfd401d741a3c819d897"
   integrity sha1-6SQ0v6XqjBn0HN/UAddBo8gZ2Jc=
 
-prettier@2.2.1, prettier@^2.0.1, prettier@~2.2.1:
+prettier@^2.0.1, prettier@~2.2.1:
```

Comment:
After reverting the prettier change in the root package.json, run `yarn` and then `npx yarn-deduplicate --packages prettier` to dedupe this.